Scripts/NuGet/EdFi.Installer.SandboxAdmin/Install-EdFiOdsSandboxAdmin.psm1

```diff
@@ -75,6 +75,10 @@ function Install-EdFiOdsSandboxAdmin {
         [string]
         $PackageVersion,
 
+        # NuGet package source . default value is set for release package source for installer .
```

Comment:
can we remove the changes that belong to the other PR

Reply:
I removed it . Thanks